model/countries/DE/DE-parsing-rules.yaml

```diff
@@ -155,3 +170,17 @@ test_parsing_definitions:
   output:
     street: "Implerstr."
     building: "15"
+- id: "Test 16"
+  type: street-address-alternative-1
+  input: "Implerstr. nummer 73 \nRückgebäude"
+  output:
+    street-address-alternative-1: "Implerstr. nummer 73 \nRückgebäude"
+    street-location: "Implerstr. nummer 73 "
```

Comment:
I guess it's fine here, but do you know whether Chrome trims pending whitespaces?

Reply:
One thing I just discovered digging into this is that `ParseStreetLocation` doesn't do well with ending spaces. Something we need to fix.